jinad/config.py

```diff
@@ -70,7 +70,7 @@ def validate_name(cls, value):
 
 class LogConfig(BaseConfig):
     # TODO: Read config from some file
-    LOG_BASE_PATH: str = '/tmp/jina-log'
+    PATH: str = '/tmp/jina-log/%s/log.log'
```

Comment:
use better 'f'{}'

Reply:
This will get replaced with `log_id` for every websocket invocation. `f-string` lazy evaluation is non-trivial, I think

Reply:
in the future we could have this configurable